packages/components/src/components/DiagramSequence.vue

```diff
@@ -283,6 +285,58 @@ export default {
 
       return '';
     },
+    collapseActionsForCompactLook() {
+      // Collapse any Actions that satisfy the following conditions:
+      //
+      // -Greater than 4 levels deep in the stack
```

Comment:
After messing around with this in various appmaps, I think it's best if we change this from 4 to 2.